### DISCUSSION TITLE
history -c command not working

### DISCUSSION THREAD

**Discussion by usgeeus** [Cyfrin/foundry-full-course-cu#1242] [General]
I want to clear history at vs code terminal
I enter the following
```
history -c
```
But I get the following Error
```
history: bad option: -c
```
How to solve this?
I am following along with this [part](https://www.youtube.com/watch?v=umepbfKp5rI). 7:24:25

**Comment by xyz899** [comment]
#### Hi,

#### `history -c` does not work on PowerShell or Windows-Based Terminals, you'll need a Unix-based terminal like zsh, wsl, or bash in order for it to work and delete your history :
##### Here I ran on WSL 2, history to see my latest commands and `history -c` and then `history`

![Screenshot 2023-12-25 103646](https://github.com/Cyfrin/foundry-full-course-f23/assets/112932882/1f58225a-7ceb-452c-bc4a-8b4b7813de81)

**Comment by Sushobhan03** [comment]
'history -c' is a bash command. Use 'history -p' if you are using zsh.
